### PR TITLE
Actually include the ruler

### PR DESCRIPTION
**Pull request by amckinley**
Not sure why this wasn't included in the example... Adds a dependency on this library and sets the default config similarly to the existing defaults.

Also updates the name of a deprecated config option.

### CODE REVIEW

File: cortex/config.libsonnet [hunk 1]
```diff
@@ -181,7 +181,7 @@
           // Setting for index cache.
           'store.index-cache-validity': '14m',  // ingester.retain-period=15m, 1m less for safety.
           'store.index-cache-read.cache.enable-fifocache': true,
-          'store.index-cache-read.fifocache.size': 102400,
+          'store.index-cache-read.fifocache.max-size-items': 102400,
```
Comment (pracucci) [inline cortex/config.libsonnet:184]: Good catch. `-store.index-cache-read.fifocache.size` is deprecated in favour of `-store.index-cache-read.fifocache.max-size-items` 👍

File: cortex/cortex.libsonnet [hunk 2]
```diff
@@ -11,6 +11,7 @@
 (import 'querier.libsonnet') +
 (import 'query-frontend.libsonnet') +
 (import 'table-manager.libsonnet') +
+(import 'ruler.libsonnet') +
```
Comment (pracucci) [inline cortex/cortex.libsonnet:14]: I think we're not ready for this yet, unless we introduce a flag to conditionally enable/disable it. @jtlisi ?

Comment (gouthamve) [reply cortex/cortex.libsonnet:14]: I think we should be now. @gotjosh can answer better. And if we're not, we should fix it.

Comment (gouthamve) [reply cortex/cortex.libsonnet:14]: After a brief discussion with Josh, we're happy to have our users run ruler by default!

Comment (gotjosh) [reply cortex/cortex.libsonnet:14]: I think it's fine, the ruler appears to be in a good state. I do agree with Marco,  given this is an "optional" service we should make it trivial for our users to disable it.

**Comment by tomwilkie** [comment]
(rebase&push to trigger CI)

**Pull request merged**